src/reducers/events/room-occupancy-reducer.js

```diff
@@ -101,7 +101,9 @@ const roomOccupancyReducer = (state = DEFAULT_STATE, action) => {
         payload.response.hasOwnProperty("data") &&
         payload.response.data.length === 1
       ) {
-        payloadEvent = payload.response.data[0];
+        const { response: data } = payload;
```

Comment:
please roll this back if not needed
add more lines of codes

Reply:
This was an eslint requirement, it wants destructuring